packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md

```diff
@@ -0,0 +1,182 @@
+
+See [Hookdeck](https://hookdeck.com/) or their [Product docs](https://hookdeck.com/docs/introduction) for further information.
+
+### Review Hookdeck dashboard
+
+The Hookdeck dashboard provides an opportunity to review all calls to your local application. See [Browsing Events](https://hookdeck.com/docs/browsing-event) for details on using this interface. With the Hookdeck utility running, use the guest account **Login URL** to access details on your hook calls. For example, your initial guest account URL appears similar to: `https://api.hookdeck.com/signin/guest?token=5sszlk2ndmkp5puy07uixrqtatkzvhwc13gctx9o3gmgczcrfu`.
+
+Each call to your local application appears in the dashboard and includes the response body, header, and other details:
+
+![A screen shot of the Hookdeck web dashboard that includes the response body, header, and other details.](/img/hookdeck-dashboard.png)
+
+### Hookdeck development workflow
+
+Hookdeck URLs are reusable and permanent (when you create a free account) and serve the entire development lifecycle&mdash;from development to staging to production. For further information on using Hookdeck, see [Development Workflow](https://hookdeck.com/docs/dev-workflow).
+
+## Create an Okta Event Hook
+
+Create the Okta Event Hook to work with your local application, which can now be exposed externally. The Event Hook must be set up and verified within your Okta Admin Console.
+
+### Set up the Event Hook
+
+1. Sign in to your [Okta org](https://login.okta.com/).
+
+2. From the Admin Console, go to **Workflow** > **Event Hooks**.
+
+3. Click **Create Event Hook**. The **Add Event Hook Endpoint** dialog box opens.
+
+4. In the **Name** field, add a unique name for the Hook (in this example, "New User Event Hook").
+
+5. In the **URL** field, add your external service URL from Hookdeck. For this example, use the **Webhook URL** highlighted after running Hookdeck, see [Run Hookdeck](#run-hookdeck). For example, your URL should appear similar to: `https://events.hookdeck.com/e/src_s8lCGfojGBPj8L0lszAZl6fD`.
+
+    >**Note:** Hookdeck generates the URL when creating the session and incorporates the local application's hook endpoint as part of the unique URL. That is, the endpoint is not explicitly defined as part of the URL.
+
+6. Leave the **Authentication field** and **Authentication secret** values blank in this example.
+
+    However, to add Basic Authentication, add the application code required at [Add Basic Authorization and Body Parsing](/docs/guides/common-hook-set-up-steps/nodejs/main/#add-basic-authorization-and-body-parsing) and then enter the following values for those fields:
+
+    * **Authentication field** = `authorization`
+
+    * **Authentication secret** = `Basic YWRtaW46c3VwZXJzZWNyZXQ=`
+
+7. In the **REQUESTS** section of the dialog box, subscribe to the Event Type you want to monitor. In this example, a user created in the Okta org: `User created`.
```

Comment:
```suggestion
7. In the **REQUESTS** section of the dialog box, subscribe to the Event Type that you want to monitor. In this example, a user created in the Okta org is `User created`.
```

Reply:
The last part was a fragment. I fixed it, but please double-check whether it's correct. Thanks.